crates/uv/src/commands/project/init.rs

```diff
@@ -765,6 +751,43 @@ fn pyproject_project_scripts(package: &PackageName, executable_name: &str, targe
     "#}
 }
 
+/// Create or update the `.python-version` file in the given path or workspace.
+async fn write_python_version_file(
+    path: &Path,
+    python_request: Option<&PythonRequest>,
+    workspace: Option<&Workspace>,
+) -> Result<()> {
+    let Some(python_request) = python_request else {
+        return Ok(());
+    };
+
+    if let Some(workspace) = workspace {
+        let version_file = workspace.python_version_file();
+        // TODO: If the version file already exists, but does not contain the requested version,
+        // should we update it? (add the version, or replace the existing one)
+        // For now, we keep the existing behavior: if the file exists, we don't update it.
+        if !version_file.path().try_exists()? {
+            version_file
+                .clone()
+                .with_versions(vec![python_request.clone()])
+                .write()
+                .await?;
+        }
```

Comment:
If the version file already exists, but does not contain the requested version, should we update it? (add the version, or replace the existing one)
For now, we keep the existing behavior: if the file exists, we don't update it.